core/notifications.py

```diff
@@ -102,21 +102,28 @@ def notify_contact_agent(contact_agent: Contact, obj):
 def notify_export_is_ready(export: Export):
     send(
         recipients=[export.user.email],
-        subject="Votre export est prêt",
+        subject="Sèves - Votre export est prêt",
         message=f"""
 Bonjour,
 
-L'export CSV que vous avez demandé est prêt, le lien pour télécharger le fichier est : {export.file.url}.
+L'export CSV que vous avez demandé est prêt, le lien pour télécharger le fichier est : {export.file.url} .
```

Comment:
```suggestion
L'export CSV que vous avez demandé est prêt, le lien pour télécharger le fichier est : {export.file.url} .
```

C'est chiant à faire mais les NBSP, ça rend tout tellement plus pro je trouve.